packages/react/src/components/DataGrid/index.stories.tsx

```diff
@@ -249,3 +250,23 @@ export const FullServerSide = () => {
     </CunninghamProvider>
   );
 };
+
+export const DataListOnly = () => {
+  const database = useMemo(
+    () =>
+      Array.from(Array(5)).map((_value, index) => ({
+        id: `list key for element ${index}`,
+        title: faker.random.word(),
+        date: faker.date.past(1).toISOString(),
+        action: <Button size={"small"}>Do it</Button>,
```

Comment:
```suggestion
<Button size="small" color="secondary">
```

( UI UX rules states that we should never have multiple primary buttons displayed in such a table )